cmd/new.go

```diff
@@ -135,19 +135,20 @@ var (
 	newBasicTemplate = `package main
 
 import (
-    "log"
+	"github.com/gofiber/fiber/v2/middleware/logger"
 
-    "github.com/gofiber/fiber/v2"
+	"github.com/gofiber/fiber/v2"
 )
 
 func main() {
-    app := fiber.New()
+	app := fiber.New()
+	app.Use(logger.New())
 
-    app.Get("/", func(c *fiber.Ctx) error {
-        return c.SendString("Hello, World!")
-    })
+	app.Get("/", func(c *fiber.Ctx) error {
+		return c.SendString("Hello, World!")
+	})
 
-    log.Fatal(app.Listen(":3000"))
+	app.Listen(":3000")
```

Comment:
The log fatal is needed else you won't see why it failed.

Reply:
ops, i forgot xd